js/leaflet-idw.js

```diff
@@ -0,0 +1,354 @@
+/*
```

Comment:
I admit that I only skimmed the documentation, but what is needed to implement this change? Do we just remove the offending files and add a note to the readme about running a git submodule command?